pcsx2/x86/Vif_Dynarec.cpp

```diff
@@ -254,11 +254,84 @@ void VifUnpackSSE_Dynarec::CompileRoutine()
 
 	pxAssume(vCL == 0);
 
+	// Need a zero register for V2_32/V3 unpacks.
+	const bool needXmmZero = (upkNum >= 8 && upkNum <= 10) || upkNum == 4;
+
+#ifdef _WIN32
+	// See SetMasks()
+	const u32 m0 = vB.mask;
+	const u32 m3 = ((m0 & 0xaaaaaaaa) >> 1) & ~m0;
+	const u32 m2 = (m0 & 0x55555555) & (~m0 >> 1);
+	// see doMaskWrite()
+	const u32 m4 = (m0 & ~((m3 << 1) | m2)) & 0x55555555;
+	const u32 m5 = ~(m2 | m3 | m4) & 0x0f0f0f0f;
+
+	int regsUsed = 2;
+	// Allocate column registers
+	if (doMask && m3)
+	{
+		colRegs[0] = xRegisterSSE(regsUsed++);
+
+		int cS = isFill ? blockSize : cycleSize;
```

Comment:
Could be const

Reply:
My latest push shall be in support of constman